doc/TESTS.rst.txt

```diff
@@ -28,51 +28,53 @@ is found in a routine, you should write a new test for that specific
 case and add it to the test suite to prevent that bug from creeping
 back in unnoticed.
 
-To run SciPy's full test suite, use the following::
+To run NumPy's full test suite, use the following::
 
-  >>> import scipy
-  >>> scipy.test()
+  >>> import numpy
+  >>> numpy.test()
 
 or from the command line::
 
   $ python runtests.py
 
-SciPy uses the testing framework from :mod:`numpy.testing`, so all
-the SciPy examples shown here are also applicable to NumPy.  NumPy's full test
-suite can be run as follows::
+NumPy's full test suite can be run as follows::
```

Comment:
To be precise: the "full" test suite is only run when `label` is `slow`. I think the adjective could be removed: "NumPy's test suite can be run ..."

I think this section could be divided with some headings:

```
Testing NumPy
------------------
NumPy can be tested in a number of ways ...

Running tests from inside python
~~~~~~~~~~~~~~~~~~~~
describe numpy.test(), mention it can be used in a non-dev 
environment to test an installed NumPy

Running tests from the command line
~~~~~~~~~~~~~~~~~~~~~~~
describe runtests.py and link to Testing Builds with (literally) `testing-builds`

Other methods of running tests
~~~~~~~~~~~~~~~~~~~

This section is the whole point of the PR - so link to as many other
testing frameworks as you feel comfortable

Adding new tests
---------------------

The rest of this page ...
```

Only a suggestion for a possible restructure, adjust as you see fit